doc/api/process.md

```diff
@@ -2013,29 +2013,57 @@ The `process.stdin` property returns a stream connected to
 stream) unless fd `0` refers to a file, in which case it is
 a [Readable][] stream.
 
+As a [Duplex][] stream, `process.stdin` can also be used in "old" mode that
+is compatible with scripts written for Node.js prior to v0.10.
+For more information see [Stream compatibility][].
+
+In "old" streams mode the `stdin` stream is paused by default, so one
+must call `process.stdin.resume()` to read from it. Note also that calling
+`process.stdin.resume()` itself would switch stream to "old" mode.
```

Comment:
I would prefer we did not mention "old" stream mode in this docs. A user can add a `'data'` event to start reading as well, it's part of the documented API.

Reply:
This text is not something I added btw, I just moved it around. I'm happy to remove it though. Please be very specific about which part to remove or any alternative wording (I am just a casual node.js user so I don't have a good idea what is appropriate)

Reply:
@mcollina PTAL

Reply:
Ok! let's keep the text for now.